packages/core/src/memdb.ts

```diff
@@ -56,6 +56,21 @@ class MemDb extends TxProcessor {
     return result
   }
 
+  private getByIdQuery (query: DocumentQuery<Doc>): Doc[] {
+    const result = []
+    if (query._id === 'string') {
+      const obj = this.objectById.get(query._id)
+      if (obj !== undefined) result.push(obj)
+    } else if (query._id !== undefined && '$in' in query._id) {
+      const ids = query._id.$in
```

Comment:
Should it also support $like here?

Reply:
No, here get object by like id impossible, if query includes _id $like, we get objects by class, then filtering

Reply:
Nope, based on code if `Object.prototype.hasOwnProperty.call(query, '_id')` it will go with this branch and will return empty every time.

Reply:
Why not updated?